frame/bridge/relayer-game/src/lib.rs

```diff
@@ -88,93 +90,110 @@ impl WeightInfo for () {}
 decl_event! {
 	pub enum Event<T, I: Instance = DefaultInstance>
 	where
-		TcBlockNumber = TcBlockNumber<T, I>,
-		GameId = GameId<TcBlockNumber<T, I>>,
+		AccountId = AccountId<T>,
+		RelayBlockId = RelayBlockId<T, I>,
 	{
-		/// A new round started. [game id, samples, mmr members]
-		NewRound(GameId, Vec<TcBlockNumber>, Vec<TcBlockNumber>),
-
-		/// A game has been settled. [game id]
-		GameOver(GameId),
-
-		/// Pending header approved. [block number, reason]
-		PendingHeaderApproved(TcBlockNumber, Vec<u8>),
-		/// Pending header rejected. [block number]
-		PendingHeaderRejected(TcBlockNumber),
+		/// A new relay parcel proposed. [relay block id, round, index, relayer]
+		RelayProposed(RelayBlockId, u32, u32, AccountId),
+		/// A new round started. [relay block id, game sample points]
+		NewRound(RelayBlockId, Vec<RelayBlockId>),
+		/// A game has been settled. [relay block id]
+		GameOver(RelayBlockId),
+		/// Pending relay parcel approved. [relay block id, reason]
+		PendingRelayParcelApproved(RelayBlockId, Vec<u8>),
+		/// Pending relay parcel rejected. [relay block id]
+		PendingRelayParcelRejected(RelayBlockId),
 	}
 }
 
 decl_error! {
 	pub enum Error for Module<T: Trait<I>, I: Instance> {
-		/// Active Game - TOO MANY
-		ActiveGameTM,
-
-		/// Can not bond with value less than usable balance.
-		InsufficientBond,
-
-		/// Pending Header - NOT FOUND
-		PendingHeaderNF,
-
-		/// Proposal - INVALID
-		ProposalI,
-
-		/// Proposal - ALREADY EXISTED
-		ProposalAE,
-
+		/// Relay Parcel - ALREADY RELAIED
+		RelayParcelAR,
 		/// Round - MISMATCHED
 		RoundMis,
-
-		/// Target Header - ALREADY CONFIRMED
-		TargetHeaderAC,
+		/// Active Games - TOO MANY
+		ActiveGamesTM,
+		/// Game - CLOSED
+		GameC,
+		/// Relay Proposal - DUPLICATED
+		RelayProposalDup,
+		/// Usable *RING* for Bond - INSUFFICIENT
+		BondIns,
+		/// Relay Proofs Quantity - INVALID
+		RelayProofsQuantityInv,
+		/// Relay Proposal - NOT EXISTED
+		RelayProposalNE,
+		/// Extended Relay Proposal - NOT EXISTED
+		ExtendedRelayProposalNE,
+		/// Previous Relay Proofs - INCOMPLETE
+		PreviousRelayProofsInc,
+		/// Pending Relay Parcel - NOT EXISTED
+		PendingRelayParcelNE,
 	}
 }
 
 decl_storage! {
 	trait Store for Module<T: Trait<I>, I: Instance = DefaultInstance> as DarwiniaRelayerGame {
-		/// All the proposals here per game
+		/// Active games' relay block ids
+		pub BlocksToRelay get(fn blocks_to_relay): Vec<RelayBlockId<T, I>>;
```

Comment:
```suggestion
		pub BlocksToResolve get(fn blocks_to_relay): Vec<RelayBlockId<T, I>>;
```

Game (断言, Relay?)
Verification Game(Dispute Resolution)